localtypings/pxtarget.d.ts

```diff
@@ -597,7 +597,10 @@ declare namespace pxt.editor {
         Standard = "",
         PythonOnly = "python-only",
         JavaScriptOnly = "javascript-only",
-        NoBlocks = "no-blocks"
+        BlocksOnly = "blocks-only",
+        NoBlocks = "no-blocks",
```

Comment:
I don't think we should expose "no-blocks". Lets remove this option.

Reply:
This was not added as part of this PR, I don't know what parts of the codebase are using this...